README.md

```diff
@@ -679,6 +680,7 @@ ssh:
       username: ubuntu  # SSH Login username
       password: xxxxx   # SSH Login password
       key: /path/to/private.key # SSH login private file
+      private_password: xxxxxxx  # PrivateKey password
```

Comment:
I think use the name `passphrases` might be better.

Reply:
me too

Reply:
> I think use the name `passphrases` might be better.

good idea